src/Models/HydrostaticFreeSurfaceModels/implicit_free_surface.jl

```diff
@@ -19,26 +19,27 @@ struct ImplicitFreeSurface{E, G, B, I, M, S}
 end
 
 """
-    ImplicitFreeSurface(; solver=:PreconditionedConjugateGradient, gravitational_acceleration=g_Earth, solver_settings...) =
+    ImplicitFreeSurface(; solver_method=:Default, gravitational_acceleration=g_Earth, solver_settings...)
```

Comment:
I wonder if there's a way to get the function signature automatically like python does, so these little typos (that you just fixed) don't happen.

Reply:
Nice point. Perhaps.

Reply:
https://juliadocs.github.io/DocStringExtensions.jl/latest/#DocStringExtensions.SIGNATURES ?

Reply:
I like this. Seems like it would at least help and bring some consistency.